lib/graphql/loadtime_type_error.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+module GraphQL
+  class LoadtimeTypeError < GraphQL::Error
```

Comment:
In other cases like this one, I've used a bare `raise` call for cases like this. 

Here's my thought: this error is only raised when something is _very_ wrong, and the user program should not continue. I can't think of any use case for rescuing this error, can you? 

If that's true, then it's important that we provide a good message (like the one you added below) and just crash with a good message. 

Adding an error class has a downside which I'd like to avoid: it adds a bit of API surface area, which I'll have to keep in mind when modifying this code. 

What do you think, would it be ok to use a plain `raise` (or `fail`, if you prefer) here?